package/network/utils/linux-atm/patches/900-fix-gcc14-build.patch

```diff
@@ -0,0 +1,82 @@
+--- a/src/arpd/io.c
++++ b/src/arpd/io.c
+@@ -615,7 +615,7 @@ int ip_itf_info(int number,uint32_t *ip,
+ int get_local(int fd,struct sockaddr_atmsvc *addr)
+ {
+     int result;
+-    size_t length;
++    socklen_t length;
```

Comment:
Interestingly enough, this is introduced by that debian patch (which has been updated btw). I wonder if it makes sense to fold this patch into that one.

Reply:
I updated the Debian patch to their latest in Sid but think it would be better to not fold it. The only advantage I could find is that it allowed me to drop `package/network/utils/linux-atm/patches/600-fix-format-errors.patch`

Reply:
Good enough.

Reply:
Also was able to drop `800-include_sockios.patch`